packages/api/src/graphql/mutation.public.ts

```diff
@@ -123,8 +124,12 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
       description: 'This mutation allows to add a comment. The input is of type CommentInput.',
       async resolve(_, {input}, {authenticateUser, dbAdapter}) {
         const {user} = authenticateUser()
+        const commentLength = countRichtextChars(0, input.text)
 
-        validateCommentLength(input.text)
+        if (commentLength > MAX_COMMENT_LENGTH) {
+          throw new CommentLengthError()
+          return
```

Comment:
the `return` is not needed. After the throw it will abort anyway until something is catching the error.